src/main/resources/amenity-data.yml

```diff
@@ -2,4 +2,23 @@ amenity-data:
   amenityConfigs:
     - amenity: scratching post
     - amenity: toys
-    - amenity: garden
\ No newline at end of file
+    - amenity: garden
```

Comment:
Zastanawiam się nad czymś bardziej generycznym, np.
- Outside Play Area
- Outdoor Open Area

Ew. można dodać, podobnie jak jest w innych, coś w stylu "Secured Outside Open Area" albo "Enclosed Outside Open Area"

Reply:
mozna

Reply:
Ja bym jednak zostawił `garden` dla psów np. bo dla psa to może mieć duże znaczenie czy ma swój własny ogród.

Reply:
To znaczy ja na froncie i tak nie przełlumacze tego jako "zewnętrzna przestrzeń do zabawy" tylko dam po prostu "ogród".